samples/.gitignore

```diff
@@ -0,0 +1 @@
+*/Properties
```

Comment:
Odd, do you see the same thing when you run the MVC Samples? We don't have this restriction there.